src/validation.ts

```diff
@@ -68,10 +68,24 @@ export class Validator {
     ) {
     }
 
+    /**
+     * Issue #117: Joi's `validate()` method dies when we pass it our own validation options, so we need to strip it
+     *  out.
+     * @url https://github.com/joiful-ts/joiful/issues/117
+     */
+    protected rectifyOptions(options: ValidationOptions | undefined): [typeof Joi, Joi.ValidationOptions?] {
```

Comment:
nit: Could we maybe name this `extractOptions` or `extractJoiOptions`? Not just because `rectify` makes me think of a `rectum`, okay mostly that. 🍑

Reply:
😆 Sure! (Originally I thought I'd also need to merge `defaultOptions`, but it wasn't necessary.)